src/sentry/conf/server.py

```diff
@@ -1409,6 +1409,8 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     "organizations:org-roles-for-teams": False,
     # Enable new JS SDK Dynamic Loader
     "organizations:js-sdk-dynamic-loader": False,
+    # If true certain Slack messages will be escaped to prevent rendering markdown
+    "organizations:slack-escape-messages": False,
```

Comment:
Why does this need to be a feature flag

Reply:
@evanpurkhiser slightly worried it's going to break formatting and I want to test internally first

Reply:
cool, let's just make sure it get's cleaned up 😆

Reply:
@evanpurkhiser lol yea well i def have to remove this once it's ready to go GA